app/src/main/res/drawable/network_nimanagua_logo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="64dp"
+    android:height="64dp"
+    android:viewportWidth="64"
+    android:viewportHeight="64">
+
+    <path
+        android:fillColor="#0067c6"
+        android:strokeWidth="0.08262365"
+        android:pathData="M 0.0000017725483 0 H 64.0000017725 V 64 H 0.0000017725483 V 0 Z" />
+    <path
+        android:fillColor="#ffffff"
+        android:strokeWidth="0.082623"
+        android:pathData="M 0 21.333668 H 64 V 42.666668 H 0 V 21.333668 Z" />
+    <path
+        android:fillColor="#c8a400"
+        android:strokeWidth="0.76141131"
+        android:pathData="M 32.835766 22.1992101 C 38.1324754817 22.1992101 42.4263069 26.4930415183 42.4263069 31.789751 C 42.4263069 37.0864604817 38.1324754817 41.3802919 32.835766 41.3802919 C 27.5390565183 41.3802919 23.2452251 37.0864604817 23.2452251 31.789751 C 23.2452251 26.4930415183 27.5390565183 22.1992101 32.835766 22.1992101 Z" />
+    <path
+        android:fillColor="#ffffff"
+        android:strokeWidth="1.2304405"
```

Comment:
Way better smaller file! You could also remove these `android:strokeWidth` from the XML. They don't do anything on Android.